lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -13,9 +13,10 @@ const ReactDom = require('react-dom');
 $(document).ready(function() {
   const campaign = Drupal.settings.dsCampaign || null;
   const user = Drupal.settings.dsUser || null;
+  const disabled_langs = ['mx', 'br']
   let localStorageKey = null;
 
-  if (campaign && user) {
+  if (campaign && user && disabled_langs.indexOf(campaign.language.language_code) == -1) {
```

Comment:
The logic im about to push up is,

localStorageKey = null
if the page has the campaign data, user data, and they're not MX/BR
set the storage key variable